create_stacksentinel_run_files.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+########################
+# Author: Sara Mirzaee
+#######################
+
+import os
+import subprocess
+import sys
+import glob
+
+import argparse
+from rsmas_logging import loglevel
+import messageRsmas
+
+from _processSteps import create_or_update_template, create_or_copy_dem
+from _process_utilities  import get_work_directory, get_project_name
```

Comment:
This is not a fully formed opinion, but it's generally best practice to import a whole module and then call its subfunctions. So here, I think we should refactor this to:

```
import _process_utilities as putils
```
and then when you use it
```
putils. get_work_directory(....
```